Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -7,16 +7,37 @@ namespace BootCamp.Chapter
     /// <summary>
     /// Caesar cipher which supports all ASCII character (256 in total)
     /// </summary>
-    public static class CaesarCipher
+    public class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            return EncryptString(message, shift);
         }
 
+        private static string EncryptString(string message, int shift)
+        {
+            if (message == null)
+            {
+                return null;
+            }
+
+            List<char> encryptString = new List<char>();
+
+            for (int j = 0; j < message.Length; j++)
+            {
+                var messageToInt = (int)message[j];
```

Comment:
You could be more clear with the way you name variable. It's not a message to int, it's a character to int. Character to int is asciiCode.